### ISSUE TITLE
AddGradleEnterprise recipe issue with existing settings plugin

### ISSUE THREAD

**Issue by alextu** [openrewrite/rewrite#2619]
Given a settings file containing a plugins block, for example:

```
plugins {
    id("org.gradle.test-retry") version("1.5.0") apply(false)
}
```

When running `org.openrewrite.gradle.plugins.AddGradleEnterprise` no change happens, instead is expected:

```
plugins {
    id("org.gradle.test-retry") version("1.5.0") apply(false)
    id 'com.gradle.enterprise' version '3.12.2'
}
```

**Comment by shanman190** [comment]
This should be fixed already with the latest snapshot builds via https://github.com/openrewrite/rewrite/commit/ca5706c63eb9fd22b95e69441771406201fc1660.

**Comment by jkschneider** [comment]
Confirmed fixed with ca5706c.

**Comment by alextu** [comment]
Thanks, I confirm that fixes it with latest SNAPSHOT
